### ISSUE TITLE
Acrobat stuck on downloading

### ISSUE THREAD

**Issue by ZweigOnly** [X1a0He/Adobe-Downloader#9]
ACC installed.

Acrobat is stuck on Downloading on adobe downloader. Files is downloaded in to Downloads folder but is not installing into apps.

**Comment by X1a0He** [comment]
> ACC installed.
> 
> 
> 
> 
> 
> Acrobat is stuck on Downloading on adobe downloader. Files is downloaded in to Downloads folder but is not installing into apps. 

After downloading Acrobat, it is a DMG file. You can manually install it, but it does not support automatic installation. Isn't it a DMG file after downloading?

**Comment by ZweigOnly** [comment]
> > ACC installed.
> > Acrobat is stuck on Downloading on adobe downloader. Files is downloaded in to Downloads folder but is not installing into apps.
> 
> After downloading Acrobat, it is a DMG file. You can manually install it, but it does not support automatic installation. Isn't it a DMG file after downloading?

That works but when manually installed the program registers as not activated.

**Comment by X1a0He** [comment]
> > > ACC installed.
> 
> > > Acrobat is stuck on Downloading on adobe downloader. Files is downloaded in to Downloads folder but is not installing into apps.
> 
> > 
> 
> > After downloading Acrobat, it is a DMG file. You can manually install it, but it does not support automatic installation. Isn't it a DMG file after downloading?
> 
> 
> 
> That works but when manually installed the program registers as not activated. 

Of course, this is only for the convenience of users who do not need to install ACC, as you can see in the readme, All Adobe apps in Adobe Downloader are from official Adobe channels and are not cracked versions.
